src/fruity/fruity-host-session.vala

```diff
@@ -790,7 +790,7 @@ namespace Frida {
 
 			try {
 				var lockdown = yield lockdown_provider.get_lockdown_client (cancellable);
-				var lldb_stream = yield lockdown.start_service (DEBUGSERVER_SERVICE_NAME, cancellable);
+				var lldb_stream = yield lockdown.start_service (DEBUGSERVER_SERVICE_NAME + "?tls=handshake-only", cancellable);

```

Comment:
Hmm, perhaps we should move these operations into a function named something like `start_lldb_service ()`:

```Vala
var lldb_stream = yield lockdown.start_service (DEBUGSERVER_SERVICE_NAME + "?tls=handshake-only", cancellable);
var lldb = yield LLDB.Client.open (lldb_stream, cancellable);
```

(I've already violated DRY pretty bad here 😅)